### PR TITLE
Newcomers_Guide: Correct the sign-up instructions

### PR DESCRIPTION
**Pull request by burrowdown**
Update instructions for signing up in gitter, now that coalabot adds
new users rather than waiting for a maintainer. Sections 1 and 2
combined, all subsequent section numbers reduced by 1.

Closes https://github.com/coala/coala/issues/4251

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by supergr35** [comment]
It looks like https://github.com/coala/coala/issues/4269 is currently being worked on by someone else, and also is already assigned to them. Can you please remove the commit referencing this issue, as they have already been assigned and are working on it?

CC: @underyx

File: docs/Developers/Newcomers_Guide.rst [hunk 1]
```diff
@@ -23,14 +23,15 @@ following tasks, you will become a developer at coala:
...
-Once you have achieved all these, just ask for being promoted on the chat and
-provide links to your reviews and merged Pull Requests. Then, you will be able
+
+When you have run coala on a project, please fill out our
+`usability survey <http://coala.io/usability>`_. Once you have successfully
+merged your first Pull Request, fill in our
+`survey form <http://coala.io/newform>`_. That will help us make your
+experience better.
+
+Once you have achieved these steps, just ask to be promoted on the chat and
```
Comment (RJ722) [inline docs/Developers/Newcomers_Guide.rst]: IMHO: `achieved` -> `completed `
because *goals* are achieved, steps should be completed. 😄

File: docs/Developers/Newcomers_Guide.rst [hunk 2]
```diff
@@ -79,32 +80,26 @@ What documentation was missing? What was hard to understand?
...
 
-Once you complete this, please take the time and
-`fill this form <https://coala.io/usability>`_ so we can improve this!
+Once you complete this, please take the time to
+`fill out this form <https://coala.io/usability>`_ so we can improve the
+process.
 
-Step 1. Meet the Community!
----------------------------
+Step 1. Meet the Community and Get an Invitation to the Organization
```
Comment (RJ722) [inline docs/Developers/Newcomers_Guide.rst]: You can add an exclamation mark at the end!

**Comment by RJ722** [comment]
Are you still working on this @burrowdown ? 😄

File: docs/Developers/Newcomers_Guide.rst [hunk 3]
```diff
@@ -82,29 +82,22 @@ What documentation was missing? What was hard to understand?
...
+`coala gitter <https://coala.io/chat>`_. The newcomers should ping us "Hello
+World" to let us know they are here because we care!
 
-Let us know on gitter that you are interested in contributing and ask for an
-invitation to our org. This is your first step towards contributing.
-A maintainer will command ``cobot`` (our gitter bot) to invite
-you and be part of the Newcomer team.
-The invitation will be sent by mail and you will have to accept
-it to join. If you don't find the invitation, accept it `here <https://github.com/coala>`__.
+When you say "Hello World" in chat ``cobot`` (our gitter bot) will invite you
```
Comment (nityeshaga) [inline docs/Developers/Newcomers_Guide.rst]: chat -> the chatroom
The line should read:
When you say "Hello World" in the chatroom, ``cobot`` (our gitter bot) will invite you

**Comment by nityeshaga** [comment]
I would like to suggest some changes in the file (the 2nd commit in this PR) for issue #4269 but I am unable to find out how to suggest those changes in it. Can someone help?

File: docs/Developers/Newcomers_Guide.rst (hunk 3 above)
Comment (nityeshaga) [inline docs/Developers/Newcomers_Guide.rst]: a line (probably line-253 in the current changed file) has a typo. Currently it reads:

> more about it, have patience and it is explained on the next step), and all

The first 'and' must be changed to 'as'. This is how it should read:

> more about it, have patience as it is explained on the next step), and all

Comment (RJ722) [inline docs/Developers/Newcomers_Guide.rst]: Nice catch @burrowdown but I think this changes is unrelated to the changes this PR is proposed for. IMHO you should open a new issue for this and label it `difficulty/newcomers` 😄

Comment (nityeshaga) [inline docs/Developers/Newcomers_Guide.rst]: @RJ722 did u mean to tag me in the above comment?

**Comment by nityeshaga** [comment]
I don't get why there is a 2nd commit on this PR that works on issue #4269 . Shouldn't one PR work on just one issue? Also #4269 is already being addressed in [another PR](https://github.com/coala/coala/pull/4271)

File: docs/Developers/Newcomers_Guide.rst (hunk 3 above)
Comment (RJ722) [inline docs/Developers/Newcomers_Guide.rst]: Yup! @nityeshaga 😄

**Comment by RJ722** [comment]
> I don't get why there is a 2nd commit on this PR that works on issue #4269 .

@burrowdown can squash the commits when this PR will be ready. 😄 

>  Shouldn't one PR work on just one issue? Also #4269 is already being addressed in another PR

There are two people assigned to #4269 I guess, that's why.

File: docs/Developers/Newcomers_Guide.rst (hunk 3 above)
Comment (nityeshaga) [inline docs/Developers/Newcomers_Guide.rst]: So should I open 2 separate issues for the 2 suggestion that I gave or just a single?

Comment (RJ722) [inline docs/Developers/Newcomers_Guide.rst]: Just commenting on #4269 would be enough, because it is already about Improving English in Newcomer's guide.

**Comment by nityeshaga** [comment]
IMHO I think that much more progress has been made in [this PR](https://github.com/coala/coala/pull/4271) with respect to #4269 . So maybe the 2nd commit should be removed. :-)

**Comment by RJ722** [comment]
commits of this PR are independent of https://github.com/coala/coala/pull/4271 so we need not worry. When **this** PR is complete, we can squash in commits to just one! 😄

**Comment by nityeshaga** [comment]
Wouldn't it be a problem if 2 PR's work on resolving the same issue?

**Comment by RJ722** [comment]
Nope! They may work on the same issue, if they try to resolve different errors though.

For example: There is an issue (`#3`) saying there are two different errors in file A: `err1` and `err2`.
Let's say there are two PR's going: `#1` and `#2`. Now, `#2` resolves `err1` and `#2` resolves `err2`. When `#1` would get merged, it may happen that `#2` faces merge conflicts and after the author of `#2` resolves them, It is good to go. 🎉 

After both the PR's are merged, we can close the issue `#3`. 🎉

**Comment by nityeshaga** [comment]
Are you still working on this @burrowdown ?

**Comment by sils** [comment]
migrated and merged for you via https://github.com/coala/coala/pull/4437
